### PR TITLE
refactor: remove unnecessary auth

### PR DESCRIPTION
**Pull request by cartcrom**
removed AWS authentication which is no longer needed after backend updates.

A private api key is no longer needed to fetch schema as well, so schema has been removed from the repo, and will be fetched automatically by yarn build now

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 # generated graphql types
 __generated__/
+schema.graphql
```
Comment (vm) [inline .gitignore:13]: why can this be ignored now?

File: fetch-schema.js [hunk 2]
```diff
@@ -8,11 +8,5 @@ const thegraphConfig = require('./relay_thegraph.config')
...
 
-const API_URL = process.env.REACT_APP_GQL_API_URL
-const API_KEY = process.env.REACT_APP_GQL_API_KEY
-
-if (API_URL && API_KEY) {
-  exec(`get-graphql-schema ${API_URL} --h X-API-KEY=${API_KEY} > ${dataConfig.schema}`)
-} else {
-  console.log('REACT_APP_GQL_API_URL or REACT_APP_GQL_API_KEY is missing from env.local')
-}
+const DATA_API_URL = 'https://api.uniswap.org/v1/graphql'
```
Comment (vm) [inline fetch-schema.js:11]: nit: should this use the process.env variable still?

**Review by vm — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (cartcrom) [reply .gitignore:13]: now it gets fetched before build

File: fetch-schema.js (hunk 2 above)
Comment (cartcrom) [reply fetch-schema.js:11]: fair point, fixed

**Pull request merged**